recaf-ui/src/main/java/software/coley/recaf/ui/control/richtext/problem/Problem.java

```diff
@@ -8,38 +8,11 @@
  *
  * @author Matt Coley
  */
-public class Problem implements Comparable<Problem> {
-	private final int line;
-	private final int column;
-	private final ProblemLevel level;
-	private final ProblemPhase phase;
-	private final String message;
+public record Problem(int line, int column, int length, ProblemLevel level, ProblemPhase phase,
```

Comment:
There are no longer javadocs for the parameters of the constructor. The old constructor docs for `@param` use need to be copied to the class docs now that this is a record.